dockerfiles/hpcc/templates/thor.yaml

```diff
@@ -30,75 +37,66 @@ spec:
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ printf "%s-slave" .name | quote }}
+  name: {{ $slaveName | quote }}
 spec:
   replicas: {{ $thor.numSlaves }}
   selector:
     matchLabels:
-      run: {{ printf "%s-slave" .name | quote }}
+      run: {{ $slaveName | quote }}
   template:
     metadata:
       labels:
-        run: {{ printf "%s-slave" .name | quote }}
+        run: {{ $slaveName | quote }}
     spec:
-      {{ if $.Values.global.singleNode | default false -}}
-      initContainers:
-      # This is a bit of a hack, to ensure that the persistent storage mounted for dllserver
-      # is writable. This is not something we would want to do if using anything other than
-      # local storage (which is only sensible on single-node systems).
-      - name: volume-mount-hack
-        image: busybox
-        command: ["sh", "-c", "chown -R 999:1000 /var/lib/HPCCSystems/hpcc-data"]
-        volumeMounts:
-        - mountPath: "/var/lib/HPCCSystems/hpcc-data"
-          name: mythorstorage
+      {{- if $.Values.global.singleNode | default false }}
```

Comment:
Leading spaces here probably best removed (they will be trimmed anyway by the {{- )

Reply:
where can indent without effect, I thought looked clearer, but I'll change